src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -489,6 +489,12 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('settingsTocVisible', "Controls whether the settings editor Table of Contents is visible."),
 			'default': true,
 			'scope': ConfigurationScope.WINDOW
+		},
+		'workbench.enableOfflineMode': {
+			'type': 'boolean',
+			'description': nls.localize('settingsOfflineMode', "Enables offline mode where no requests are made over the network by VS Code."),
```

Comment:
I wonder if this wording should be changed to say that no requests are made *to Microsoft*, as we do still make requests to SCM providers, and there's nothing we can do to stop extensions (so far as I know)